doc/using-ray-on-a-cluster.md

```diff
@@ -97,41 +97,30 @@ block until the installation has completed. The standard output from the nodes
 will be redirected to your terminal.
 5. To check that the installation succeeded, you can ssh to each node, cd into
 the directory `ray/test/`, and run the tests (e.g., `python runtest.py`).
-6. Create a directory (for example, `mkdir ~/example_ray_code`) containing the
-worker `worker.py` code along with the code for any modules imported by
-`worker.py`. For example,
-
-    ```
-    cp ray/scripts/default_worker.py ~/example_ray_code/worker.py
-    cp ray/scripts/example_functions.py ~/example_ray_code/
-    ```
-
-7. Start the cluster (the scheduler, object stores, and workers) with the
-command `cluster.start_ray("~/example_ray_code")`, where the second argument is
-the local path to the worker code that you would like to use. This command will
-copy the worker code to each node and will start the cluster. After completing
-successfully, this command will print out a command that can be run on the head
-node to attach a shell (the driver) to the cluster. For example,
+6. Start the cluster (the scheduler, object stores, and workers) with the
+command `cluster.start_ray("~/example_ray_code")`, where the argument is
+the local path to the directory that contains your Python code. This command will
+copy this source code to each node and will start the cluster. Each worker that
+is started will have a local copy of the ~/example_ray_code directory in their
+PYTHONPATH. After completing successfully, this command will print out a command
```

Comment:
Yeah, it's the user-specified `PYTHONPATH` added to the default ones from the installation.